docs/tutorials/create-waggle.md

```diff
@@ -1,37 +1,28 @@
+During the bootstrapping process you will have the option to register your device within the [web portal](https://portal.sagecontinuum.org/apps/explore). It is highly recommended to register your device, as this enables all the core [WES tools](LINK HERE) to be automatically downloaded, enabling the edge app development and run-time environment. Additionally, this enables your edge apps to publish data to the development [Beehive](LINK HERE), accessible to cloud-based analysis tools and workflow frameworks.
 
-An inexpensive Sage Edge Computing kit that can enable students to run existing AI@Edge computations with cameras and microphones, and then develop their own codes. The Sage Beehive will host the data and make it available to cloud-based analysis tools and workflow frameworks. This document describes the overall concept, some initial goals, and workflow.
+To register your device navigate to the [My Devices Web Portal](https://portal.sagecontinuum.org/my-devices). Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. A "registration zip" file will be generated and available for download. Then follow the instructions for [your device](https://github.com/waggle-sensor/node-platforms) to load the registration keys.
 
-## Support Platforms
+> You may register as many times as you want. But note that each registration key has a short expiration time and should be used shortly after generation.
 
-The only supported platform now is Nvidia Nano, please refer to the [link](https://github.com/waggle-sensor/node-platforms/tree/main/nvidia-nano) to see the guide of setting up Nvidia Nano.
+Now you are ready to develop your edge apps and/or introduce new sensors to the Waggle platform. Head over to the [overview](../about/overview.md) to find the instructions you need for development.
 
-In the future, we will support more [platforms](https://github.com/waggle-sensor/node-platforms) such as Raspberry Pi 4, Wild Waggle Node, etc. You can also design your own device and follow the steps below to register to the platform.
 
-## Register Your Own Device
-
-In order to register your waggle device, you will need keys to authenticate your device with beehive when setting up your nano device. To get registration keys, please log in to [Sage Portal](https://portal.sagecontinuum.org/apps/explore) with your Globus credentials, after logging in, visit [My devices](https://portal.sagecontinuum.org/my-devices) page under your user account. Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. It will generate a zip file for you. Check your downloads folder for the zip file, then move it to the same folder with the ansible script to proceed the execution.
-
-![Figure 1: Sage Portal - My devices](./images/sage-my-devices.png)
\ No newline at end of file
+![Figure 1: Registering my devices](./images/sage-my-devices.png)
```

Comment:
Move image above the note (`> You may register as many`) above to git the image some context